### ISSUE TITLE
There is an error in the forward function of the PointPromptTraining class.

### ISSUE THREAD

**Issue by yanqingbing888888** [Pointcept/Pointcept#476]
@MODELS.register_module("PPT-v1m3")
class PointPromptTraining(nn.Module):
      def __init__(
          self,
          ...
          conditions=("Structured3D", "ScanNet", "S3DIS"),
          ...
      ):
        ...
        self.register_buffer("class_embeddings", class_embeddings, persistent=False)
        ...

    def forward(self, data_dict):
        condition = data_dict["condition"][0]
        ...
        sim = (
            feat
            @ self.class_embeddings.split(self.num_classes)[
                self.conditions.index(condition)
            ].t()
        )
Note: 
   The line of code above has an error. In this case, only the class corresponding to condition("Structured3D") will be calculated for similarity. The classes corresponding to the "ScanNet" and "S3DIS" datasets will not be calculated for similarity. Therefore, they will not be involved in the calculation of the loss function. As a result, the "ScanNet" and "S3DIS" datasets will not play any role in the training process.
